### PR TITLE
Fix recently introduced grammatical errors

### PR DESCRIPTION
**Pull request by CandyAngel**
### Summary
This fixes the grammatical errors recently introduced into the documentation.

### References
ether mentioned this in IRC.

### CODE REVIEW

**Comment by kraih** [comment]
You made it inconsistent. 👎

**Comment by Grinnz** [comment]
They could all be changed to "This is used..."

**Comment by CandyAngel** [comment]
I've separated the implementation details to another paragraph so the first part can be consistent with other ", used for" usage.

File: lib/Test/Mojo.pm [hunk 1]
```diff
@@ -1137,10 +1137,10 @@ Opposite of L</"status_is">.
...
   $t = $t->test('is', 'first value', 'second value', 'right value');
 
-Use L<Test::More> functions such as C<is>, C<ok> and C<is_deeply> safely
-(telling L<Test::Builder> how far up the call stack to look when something
-fails) through L</"handler"> and store the result in L</"success">. Used to
-implement L<Test::Mojo> roles.
+Call L<Test::More> functions safely, used to implement L<Test::Mojo> roles.
+
+Calls are made through L</"handler"> (after telling L<Test::Builder> how far
+up the call stack to look on failure) with the result stored in L</"success">.
```
Comment (kraih) [inline lib/Test/Mojo.pm:1143]: At this point you might as well just remove the whole paragraph.

**Pull request merged**